assets/js/infinitescroll.js

```diff
@@ -34,7 +33,29 @@ $(function ($) {
         ticking = true;
     }
 
-    function infiniteScroll() {
+    function sanitizePathname(path) {
+        var paginationRegex = /(?:page\/)(\d)(?:\/)$/i;
+
+        // remove hash params from path
+        path = path.replace(/#(.*)$/g, '').replace('////g', '/');
```

Comment:
The original regex for double slash was `/\//g`, which was targeting only a single slash, but according to the commit message [here](https://github.com/TryGhost/Casper/pull/325#issue-129652376) it was supposed to replace double slashes.